url-detector/src/main/java/com/linkedin/urls/detection/DomainNameReader.java

```diff
@@ -320,6 +320,18 @@ public ReaderNextState readDomainName() {
 
     //while not done and not end of string keep reading.
     boolean done = false;
+    boolean isAllHexSoFar = (_current == null || _current.equals("")) &&
+      _reader.canReadChars(3) &&
+      ("0x".equalsIgnoreCase(_reader.peek(2)));
+
+    if (isAllHexSoFar) {
+      //append hexa radix symbol characters
```

Comment:
Add space after the `//`
Change `append` to `Append`

Reply:
The convention here is not to add space.
Changing to uppercase